code/modules/projectiles/projectile/beams.dm

```diff
@@ -46,25 +46,31 @@ In pvp they also have more lasting damages, such as infections, and pain form bu
 
 /obj/item/projectile/beam/weak/pistol_35
 	damage_types = list(BURN = 12)
+	armor_penetration = 5
```

Comment:
Why are you nerfing the pistol one?

Reply:
Adjusted to AP of standard. Was feeling odd about it myself and could potentially be balanced out with reducing the material cost of 9mm laser.

Another fact about it being 15 was that it would have the same performance rating as other larger calibers making it a bit awkward.